setup.py

```diff
@@ -7,23 +7,79 @@
 # ----------------------------------------------------------------------------
 
 from setuptools import setup, find_packages
+from setuptools.command.install import install
+from setuptools.command.develop import develop
 
 import versioneer
 
+
+def _post(obj):
+    import urllib.request
+    import shutil
+    import os
+
+    assets_dir = os.path.join(obj.install_libbase,
+                              'q2_american_gut/assets/')
+
+    if not os.path.exists(assets_dir):
+        os.mkdir(assets_dir)
+
+    version = versioneer.get_version()
+    if 'dirty' in version:
+        # we're operating in dev
+        version = '2018.2'
+
+    # source the taxonomy classifier (for assignments for deblur sequences)
+    src_url = 'https://data.qiime2.org/%s/common/gg-13-8-99-515-806-nb-classifier.qza' % version  # noqa
```

Comment:
This string and the other one can be broken down into two lines, no need for the `# noqa` directive.

Reply:
done